### DISCUSSION TITLE
namelist in regtest

### DISCUSSION THREAD

**Discussion by mickaelaccensi** [NOAA-EMC/WW3#572] [General]
Hi folks,
In order to keep nml files content consistent in all the regtests, I would suggest to remove all the comments instead of updating them every time a new feature is added (which is not always done actually..). 

Since the default value is defined in the code, only a few regtests nml files should be modified manually to add the user defined value related to the new feature. The comment related to the new feature would only be visible in model/nml.

what's your opinion ? If you agree with that, I'll make a dedicated branch once all ww3 codes will have their own nml file (still some work to be done :/ )

**Comment by ukmo-ccbunney** [comment]
I think this is a very good idea @mickaelaccensi; there are a lot of superfluous comments in the regtests inp/nml files!

> instead of updating them every time a new feature is added (which is not always done actually..).

Oops - this makes me realise I did not update them in my last PR 🙄 Case in point...

**Comment by JessicaMeixner-NOAA** [comment]
@mickaelaccensi I agree 
I think we should add all the defaults and comments in the model/nml example files, but remove the extra from the regression tests where it makes sense (and not to systematically add them when changes are made).   I think some of that could even be implemented now, although waiting until we have all the nml available also works.
